core/src/main/java/psiprobe/tools/AccessorFactory.java

```diff
@@ -43,7 +43,13 @@ public static Accessor getInstance() {
   private static Accessor getReflective() {
     try {
       return new ReflectiveAccessor();
-    } catch (Exception ex) {
+    } catch (ClassNotFoundException e) {
```

Comment:
Perfect!  Java 1.7 it is.  Tomcat 7 does require that if wanting to use websockets.  I thought I asked that out on the google groups but didn't hear anything back.  I'll check again to be certain.  Once we make that jump...I see the code immediately taking full advantage so there won't be any going back.  Only issue is we will be jumping from 5 to 7 without any java 6 support at all.  Any concern there?